lib/vite-base-config.js

```diff
@@ -0,0 +1,45 @@
+module.exports = ({
+  mode, base, root, cacheDir, manifestRelPath, sourcemaps
```

Comment:
Even if the vite config is `sourcemap`, I suggest to keep the same camel case pattern for all our options

```suggestion
  mode, base, root, cacheDir, manifestRelPath, sourceMaps
```